src/generic_ui/polymer/settings.ts

```diff
@@ -46,7 +33,7 @@ Polymer({
     this.$.confirmNewServer.hidden = false;
   },
   resetStunServers: function() {
-    browserified_exports.model.globalSettings.stunServers = this.DEFAULT_STUN_SERVERS;
+    browserified_exports.model.globalSettings.stunServers = [];
```

Comment:
That only resets the stun servers when loadSettings is called, but I don't think that is called except when you restart the uProxy core.  So if you reset, and then immediately try to proxy again, won't you stun servers be empty?